cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -29,9 +29,13 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         |""".stripMargin)
     section("Introduction", "administration-databases-introduction") {
       p(
-        """Neo4j allows the same server to manage multiple databases. The metadata for these databases,
-          |including the associated security model, is maintained in a special database called the `system` database.
-          |All multi-database administrative commands need to be executing against the `system` database.""".stripMargin)
+        """Neo4j supports the management of multiple databases within the same DBMS.
+          |The metadata for these databases, including the associated security model, is maintained in a special database called the `system` database.
+          |Most administrative commands must be executed against the `system` database because they involve editing the metadata for the entire system.
+          |This includes all commands related to managing multiple databases as well as all commands for defining the security model: users, roles and privileges.
+          |The administrative commands that are specific to the schema of an individual database are still executed against that specific database.
+          |These include index and constraint management commands.
+          |When connected to the DBMS over bolt, administrative commands are automatically routed to the `system` database.""".stripMargin)
```

Comment:
I think this section is more an introduction to ALL administration commands, Database, Index, Constraint and Security and not just the Database where you have moved it (might be good to confirm this move with (current) Team Cypher Implementation if you think it fits better here)